### PR TITLE
update instance types

### PR DESCRIPTION
**Pull request by xsuchy**
both has been recently upgrade to meet the demand

### CODE REVIEW

File: group_vars/prod.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ keygen:
   image: f39
   old_instance_id: i-0efff0067789ed7a9
   new_instance_id: i-01b4f20bd9fb1bea6
-  instance_type: t3a.xlarge
+  instance_type: c7a.4xlarge
```
Comment (praiskup) [inline group_vars/prod.yml:88]: This isn't supposed to stay, I think?

Comment (xsuchy) [reply group_vars/prod.yml:88]: I do not know. :) But it is a reality now. We can change it later when we change it back in AWS.

Comment (praiskup) [reply group_vars/prod.yml:88]: We should probably move to c7a indeed, though not `4xlarge`, that's just too big ... namely after releasing 
https://github.com/fedora-copr/copr/pull/3052

Comment (praiskup) [reply group_vars/prod.yml:88]: > We can change it later when we change it back in AWS.

There's high chance we will forget about it :-) and we will start too-expensive instance once migrating to F41.  I'd prefer either fixing this directly to `c7a.xlarge`, or wait with this PR till we have https://github.com/fedora-copr/copr/pull/3052 deployed in prod.

Comment (xsuchy) [reply group_vars/prod.yml:88]: I think we will not forget. 

But if you insist I will either remove or change it to xlarge.

Comment (praiskup) [reply group_vars/prod.yml:88]: Yes, please specify there the expected value.  I think it should be `c7a.xlarge`.

**Comment by xsuchy** [comment]
Updated.

**Review by praiskup — APPROVED** [review]

**Comment by praiskup** [comment]
Thank you!

**Pull request merged**